.gitignore

```diff
@@ -148,3 +148,6 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/l
+
+# Flask's environment variables file
+.flaskenv
```

Comment:
Also, I'm sorry that I've not been the best at keeping everyone updated about this new way to do things.

I went for the "dotenv" approach because resetting environment variables every time I fired up a terminal was a pain and that was a suggested way in the Flask docs to solve that problem. And then I deleted `VAULT_PASSWORD` from the dev shell scripts because it was causing problems for me that conflicted with the `.env` file (it was setting the VAULT_PASSWORD to `""` instead of ignoring it if `VAULT_PASSWORD` was not previously set in the actual environment).

My original reluctance with `.env` was that I was trying to avoid it being stored in a file as we had 2 credentials leaks early into our development, but I think our current devs (that includes you!) are being very responsible with credentials, so I believe we can all handle the `.env` approach now.